deps/CVC4/include/cvc4_private.h

```diff
@@ -0,0 +1,55 @@
+/*********************                                                        */
+/*! \file cvc4_public.h

```

Comment:
IIRC, we modified this file. It was a hack. 
Now the file header and its header guard do not match with the file name. 
Maybe, we should try to have a clean solution for this.

Reply:
Good catch, will fix!